client/src/components/Banner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Button} from 'lucid-ui';
+import {Link} from "react-router-dom";
+
+
+
+function signOut(){
+  localStorage.setItem("username", "");
+  window.location.href= "/login";
```

Comment:
Yeah dont do this please, This actually doesnt push on to the state of react router 

```    this.props.history.push('/')	```